gamestates.js

```diff
@@ -9,109 +9,154 @@
  */
 let state = {
     menuState: {
```

Comment:
a lot of the "changes" from here are just automatic indents to make the code easier to read.